sys/net/gnrc/network_layer/ipv6/nib/_nib-internal.c

```diff
@@ -483,6 +483,41 @@ _nib_offl_entry_t *_nib_offl_iter(const _nib_offl_entry_t *last)
     return NULL;
 }
 
+_nib_offl_entry_t *_nib_pl_add(unsigned iface,
+                               const ipv6_addr_t *pfx,
+                               unsigned pfx_len,
+                               uint32_t valid_ltime,
+                               uint32_t pref_ltime)
+{
+    uint32_t now;
+    _nib_offl_entry_t *dst = _nib_offl_add(NULL, iface, pfx, pfx_len, _PL);
+
+    if (dst == NULL) {
```

Comment:
wow; early exit (:

Reply:
Never said I'm strictly against them :P But for `gnrc_ndp` it was pointed out by @OlegHahm so I kept it in `gnrc_ndp2` ;-).